api_app/analyzers_manager/observable_analyzers/cyberchef.py

```diff
@@ -0,0 +1,55 @@
+# This file is a part of IntelOwl https://github.com/intelowlproject/IntelOwl
+# See the file 'LICENSE' for copying permission.
+import json
+
+import requests
+
+from api_app.analyzers_manager.classes import DockerBasedAnalyzer, ObservableAnalyzer
+from api_app.exceptions import AnalyzerRunException
+
+
+class CyberChef(ObservableAnalyzer, DockerBasedAnalyzer):
+    name: str = "CyberChefServer"
+    url: str = "http://cyberchef-server:3000/bake"
+    config_filename: str = "cyberchef_recipes.json"
+
+    def set_params(self, params):
+        self.recipe_name = params.get("recipe_name", "")
+        if self.recipe_name:
+            try:
+                try:
+                    with open(f"configuration/{self.config_filename}", "r") as recipes:
```

Comment:
please use absolute path (leverage PROJECT_LOCATION variable) then we are good

Reply:
Done